.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
[You can add config to package.json instead of a separate file](https://eslint.org/docs/latest/use/configure/configuration-files), saves you from having two files, it would look similar to this:

<img width="589" alt="Screenshot 2024-04-03 at 07 31 32" src="https://github.com/ONSdigital/eq-questionnaire-launcher/assets/53475968/3936338e-4fee-4678-a2fa-34dee380879b">

Reply:
Makes sense, updated 👍